### PR TITLE
Summer 2018 Fanart Contest newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
Anyone who comments any variant of 'IT'S THE SEA' is cursed to never find a saucepan when they need it for the rest of their lives.

### CODE REVIEW

File: news/2018-07-18-summer-2018-fanart-contest.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Mocha might've stolen the show last year, but now she's gotta fight for her spot at the front! With summer in full swing, the latest season-themed fanart contest is underway, and we want to see your best!
+
+[![](https://assets.ppy.sh/contests/66/header.jpg)](https://osu.ppy.sh/community/contests/66)
+
+Whip out your drawing implement of choice and pen **any of the osu! or osu! community mascots in a summer-themed piece** and you could earn yourself fame, glory, and some sweet prizes!
+
+Those willing to brave the heated deadline will find themselves with **14 days from the date of this post** to put drawing thing to drawing medium in a sizzling display of artistic talent.
+
+Once submissions close, all the scorching submissions will placed up for **7 days of blazing voting**, with a top selection of the most highly voted entries netting their artists **2 months of osu!supporter, and also appearing in the game client as a seasonal background for several weeks.**
```
Comment (Xinnoh) [inline news/2018-07-18-summer-2018-fanart-contest.md]: will placed -> will be placed

File: news/2018-07-18-summer-2018-fanart-contest.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+* **SUBMIT ONLY YOUR OWN WORK! NO TRACING! NO REDRAWING!** It hurts that we have to keep saying this but please, we want to see *your* work, not a rehash of pixiv or DeviantArt's latest and greatest Zero Two fanart redrawn to look like an osu! mascot. We will invariably find out if you do this, and repeat offenders will be unwelcome to participate in the future.
+* **Don't use copyrighted characters from other intellectual properties.** You can freely use any of the osu! mascots (pippi, Yuzu, Mocha, Maria) and any of the community mascots (Tama, Aiko, Taikonator, so on), but stay away from using direct representations of characters from other worlds or franchises in your work. Remember that the community mascots are also someone else's creation, so make sure to treat them with the same respect that you'd like your work treated with.
```
Comment (Xinnoh) [inline news/2018-07-18-summer-2018-fanart-contest.md]: your work treated with. -> your work treated.

File: news/2018-07-18-summer-2018-fanart-contest.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+[![](https://assets.ppy.sh/contests/66/header.jpg)](https://osu.ppy.sh/community/contests/66)
+
+Whip out your drawing implement of choice and pen **any of the osu! or osu! community mascots in a summer-themed piece** and you could earn yourself fame, glory, and some sweet prizes!
+
+Those willing to brave the heated deadline will find themselves with **14 days from the date of this post** to put drawing thing to drawing medium in a sizzling display of artistic talent.
+
+Once submissions close, all the scorching submissions will be placed up for **7 days of blazing voting**, with a top selection of the most highly voted entries netting their artists **2 months of osu!supporter, and also appearing in the game client as a seasonal background for several weeks.**
+
+This also neatly lines up with the start of the osu!mania 4K World Cup, which means that the winning pieces will get a HIDDEN EXTRA PRIZE as the waiting-screen backgrounds of choice for all those who tune in to see some of osu!'s best duke it out. Spicy.
```
Comment (Joehuu) [inline news/2018-07-18-summer-2018-fanart-contest.md]: Add `2018` after `World Cup`.

File: news/2018-07-18-summer-2018-fanart-contest.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+* **SUBMIT ONLY YOUR OWN WORK! NO TRACING! NO REDRAWING!** It hurts that we have to keep saying this but please, we want to see *your* work, not a rehash of pixiv or DeviantArt's latest and greatest Zero Two fanart redrawn to look like an osu! mascot. We will invariably find out if you do this, and repeat offenders will be unwelcome to participate in the future.
+* **Don't use copyrighted characters from other intellectual properties.** You can freely use any of the osu! mascots (pippi, Yuzu, Mocha, Maria) and any of the community mascots (Tama, Aiko, Taikonator, so on), but stay away from using direct representations of characters from other worlds or franchises in your work. Remember that the community mascots are also someone else's creation, so make sure to treat their work with the same respect that you'd treat yours.
```
[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2018-07-18-summer-2018-fanart-contest.md]: good idea!

**Pull request merged**